### ISSUE TITLE
planning: Cortex Engines Management integrates with Jan data folders

### ISSUE THREAD

**Issue by gabrielle-ong** [janhq/cortex.cpp#1660]
**Problem**
- the new cortex engines management broke Jan 0.5.8 (works successfully on cortex v1.0.2-rc4)
- We will need to handle Jan's data folders to integrate with the upcoming changes for Cortex Engine Management
- @louis-jan and @namchuai could you add your thoughts here?

Related to 
#1416 
#1638 

**Success Criteria**

**Tasklist**
- [ ]

**Comment by louis-jan** [comment]
Hi @gabrielle-ong I think we can close this issue. I'm filing a new issue from Jan here since it's easier to do from Jan for now. (E.g. packaging)

https://github.com/janhq/jan/issues/4025

- Jan will download engine artifacts and structure them accordingly on CI when building the app.
- Jan sets a default engine variant on load and settings changes.

This fixes from cortex.cpp, unblocked Jan integration already.
https://github.com/janhq/cortex.cpp/pull/1687

**Comment by gabrielle-ong** [comment]
Thank you @louis-jan! Closing this issue
